src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.html

```diff
@@ -47,6 +47,7 @@ <h2>
                 *ngIf="question.type === MULTIPLE_CHOICE"
                 [question]="question"
                 [(selectedAnswerOptions)]="selectedAnswerOptions[question.id]"
+                [fnOnSelection]="onSelectionChanged.bind(this)"
```

Comment:
Why do we need `.bind(this)` here? Is `onSelectionChanged()` not sufficient?

Reply:
just copied from the other part. As it is working, I don't want to touch this ;-)